_pages/location.md

```diff
@@ -0,0 +1,319 @@
+  <header class="text-center">
+    <h2>The City</h2>
+  </header>
+    <div class="row column">
+      <div class="medium-6 column">
+      <img
+        class="thumbnail thumbnail-block"
+        src="/static/img/locations/calton-hill.jpg"
+        alt="Calton Hill" />
+      </div>
+      <div class="medium-6 column">
+        <p>
+        Edinburgh is Scotland’s capital city, and the most visited city in the UK after
+        London.  It is perhaps best known as the home of the Edinburgh Fringe and 
+        International Festivals in August each year, but there is plenty to experience at 
+        any time of year.  Many attractions and landmarks are within a few minutes walk of 
+        the conference venue, including Princes Street Gardens, the Scott Monument, the 
+        historic Old Town, the monuments and observatory at Calton Hill, and of course 
+        Edinburgh Castle and the Royal Mile.
+        </p>
+      </div>
+      <div class="column">  
+        <p>Check out the following resources for some guides to the city:
+        <ul>
+          <li><a href="https://www.visitscotland.com/destinations-maps/edinburgh/">Visit Scotland</a></li>
+          <li><a href="https://www.introducingedinburgh.com/">Introducing Edinburgh</a></li>
+          <li><a href="https://edinburgh.org/">Forever Edinburgh</a></li>
+          <li><a href="https://www.timeout.com/edinburgh">Timeout</a></li>
+        </ul>
+        </p>  
+      </div>
+    </div>  
+  </section>
+
+  <section id="weather">
+  <header class="subpage-header">
+    <h2>Weather</h2>
+  </header>
+  <div class="row column">
+    <p>
+      Edinburgh’s weather is reliably unpredictable.  May/June are typically some of the 
+      warmer and drier months of the year, but you should be prepared for any weather!  
+      It’s a good idea to have lightweight waterproof clothes with you, no matter how nice 
+      it looks first thing in the morning!
+      </p>  
+  </div>
+  </section>
+
+
+  <section id="getting-here">
+  <header class="subpage-header">
+    <h2>Getting here</h2>
+  </header>
+  <div class="row column">
+    <h3>By Air</h3>
+    <p>
+      Edinburgh Airport is just 6 miles from the centre, with direct flights to many 
+      destinations around the world. You can easily get to the centre of the city in 
+      around 20 minutes by taxi, bus & tram. You can find more information on getting into 
+      the city from the airport on the <a href="http://www.edinburghairport.com/">airport website</a>.
+    </p>
+    <h3>By Train</h3>
+    <p>
+      There are two train stations in the centre of the city, Edinburgh Waverley, and 
+      Edinburgh Haymarket. It is approximately a 10 minute walk to the Assembly Rooms from 
+      Waverley, and 20 minutes from Haymarket. If you're travelling by train it's much 
+      cheaper to book in advance. You can book tickets online on the <a href="https://
+      thetrainline.com">Trainline</a>.
```

Comment:
The trainline link does not work in preview. I think there might have been linebreak and whitespaces mixed in before "thetrainline.com"